lib/constants/calendar_settings.dart

```diff
@@ -48,5 +48,6 @@ class CalendarSettings {
     this.rowHeight = 60.0,
     this.monthSelectedColor = const Color(0xFF0474BB),
     this.dayNameFormat = "dd MMM",
+    this.minimumHeight = 0.0,
```

Comment:
I would change it to `minimumEventHeight`, otherwise its hard to know if it means whole calendar height or only event.